recipes/bump2version/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "bump2version" %}
+{% set version = "0.5.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bc169d268e616dd4aa578ad7d0da99217d037d459b0eea14387d9e4f3cdabb02
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - bumpversion = bumpversion:main
+    - bump2version = bumpversion:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  # Cannot run tests since they're not installed with the package and
```

Comment:
If they are in the tarball and you want to run them you can do something like:

https://github.com/conda-forge/compliance-checker-feedstock/blob/master/recipe/meta.yaml#L39-L49

But usually an import test is enough. I'll just ask you to remove those comments in that case. (It can be done the feedstock.)

Reply:
Done, hope that's OK now!